pkg/triggers/helper.go

```diff
@@ -34,6 +37,35 @@ func CreateHTTPClient() *http.Client {
 	return client
 }
 
+// CreateHTTPSClient for connection re-use
+func CreateHTTPSClient() *http.Client {
+	gopath := os.Getenv("GOPATH")
+	// Load client cert
+	cert, err := tls.LoadX509KeyPair(gopath+"/src/github.com/openshift-pipelines/release-tests/testdata/triggers/certs/server.crt",
```

Comment:
it's good to use relative path if possible

Reply:
modified :+1: